README.md

```diff
@@ -1,47 +1,39 @@
 
-This repository contains Netlify's API definition in the [Open API format](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md), formerly known as Swagger.
+This repository contains Netlify's API definition in the [Open API format](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) (AKA Swagger).
 
-It's still a work in progress and we welcome feedback and contributions.
+The spec is currently incomplete but includes many common endpoints.
 
-## Go Client
+## Usage
 
-We use [go-swagger](https://github.com/go-swagger/go-swagger) to validate our spec against the 2.0 spec of Open API.
+The `swagger.yml` file is the master copy of the Open API 2.0 definition.  Additional context on using the API can be found on our [docs site](https://www.netlify.com/docs/api/).
 
-We currently depend on version 0.16.0 of the swagger toolchain. You can download the binary for your platform from this release page:
+The spec is published and versioned for various ecosystems:
 
-https://github.com/go-swagger/go-swagger/releases/tag/0.16.0
+### SwaggerUI (Web UI)
 
-### Spec validation
+You can view the definition using [Swagger UI](https://swagger.io/tools/swagger-ui/) by visiting [open-api.netlify.com](http://open-api.netlify.com) which provides limited interaction with the API from the browser.
 
-You can run this command to validate the spec:
+[![screenshot of netlify swagger ui](ui/screenshot.png)](https://open-api.netlify.com)
 
-	make validate
+### Go Client
 
-### Code generation
+[![GoDoc](https://godoc.org/github.com/netlify/open-api/go?status.svg)](https://godoc.org/github.com/netlify/open-api/go) [![Go Report Card](https://goreportcard.com/badge/github.com/netlify/open-api)](https://goreportcard.com/report/github.com/netlify/open-api) [![Github release](https://img.shields.io/github/release/netlify/open-api.svg)](https://github.com/netlify/open-api/releases/latest)
 
-Currently, we're generating client code for Go, but we're planning on releasing libraries in any language that can generate code from the spec.
-
-You can use this command to generate the Go client:
-
-	make generate
-	
-You may first want to edit swagger.yml to add your field or endpoint definitions.
-
-## JS Client
+```console
+$ go get github.com/netlify/open-api/...
+```
 
-Swagger's JS client can dynamically create a client from a spec either from a URL or spec object.
+- [Porcelain](https://godoc.org/github.com/netlify/open-api/go/porcelain): High level interactions and operations
+- [Plumbing](https://godoc.org/github.com/netlify/open-api/go/plumbing): Low level client operations
+- [Models](https://godoc.org/github.com/netlify/open-api/go/porcelain): Models generated by [go-swagger]()
 
-See [swagger-js](https://github.com/swagger-api/swagger-js)
+See [CONTRIBUTING.md#go-client](CONTRIBUTING.md) for details on how this client is developed and generated.
```

Comment:
do we need `#go-client` here (looks like it's not really pointing to anywhere specifically in contributing)?

Reply:
Fixing